compute/resource_job.go

```diff
@@ -221,7 +221,12 @@ var jobSchema = common.StructToSchema(JobSettings{},
 			v.DiffSuppressFunc = common.MakeEmptyBlockSuppressFunc("new_cluster.0.gcp_attributes.#")
 		}
 		s["email_notifications"].DiffSuppressFunc = common.MakeEmptyBlockSuppressFunc("email_notifications.#")
-		s["max_concurrent_runs"].ValidateDiagFunc = validation.ToDiagFunc(validation.IntAtLeast(1))
+		s["max_concurrent_runs"] = &schema.Schema{
```

Comment:
Please don't overwrite the entire field, as this is fragile. `s["max_concurrent_runs"].Default = 1` is the less invasive way. And please mention it in doc file as well :)

P.S.: url field on the next statement is added, not overridden.